test/functional/legacy/013_autocommands_on_close_spec.lua

```diff
@@ -0,0 +1,108 @@
+-- Tests for autocommands on :close command
```

Comment:
You could perhaps preserve the paragraph structure of the original, though I don't know whether it is significant.